### ISSUE TITLE
Expanding Distributed TLS minio with new Server Pool needs explicit port in Expansion URL 

### ISSUE THREAD

**Issue by aglahe** [minio/minio#14089]
Disclaimer: This could be just a docs issue when using TLS..but, just in case it's not.

If you expand a TLS minio deployment by adding a server pool, if you set the port in the startup cmd line, you must make sure that when adding a second server pool expansion url, to include the port.  If you don't, even though the first server pool with TLS worked fine without, when adding a second, it appears as though the machines in server pool 1 will try and reach server pool 2 machines on port :443 because of the https/TLS in the URL.

## Expected Behavior
When restarting minio with additional server pool expansion URLs, it would use the default port :9000 when using TLS

ServerPool 1 start:
server --address :9000 https://minio-test1.test.local/data{1...4}

ServerPool 1 & 2 start:
server --address :9000 https://minio-test1.test.local/data{1...4} https://minio-test2.test.local/data{1...4}

```$ mc admin info debug
●  minio-test1.test.local
   Uptime: 1 minute
   Version: 2022-01-08T03:11:54Z
   Network: 2/2 OK
   Drives: 4/4 OK

●  minio-test2.test.local
   Uptime: 1 minute
   Version: 2022-01-08T03:11:54Z
   Network: 2/2 OK
   Drives: 4/4 OK
```

## Current Behavior
```
$ mc admin info debug
●  minio-test1.test.local
   Uptime: 1 minute
   Version: 2022-01-08T03:11:54Z
   Network: 2/2 OK
   Drives: 4/4 OK

●  minio-test1.test.local
   Uptime: offline
   Drives: 0/4 OK

●  minio-test2.test.local
   Uptime: 1 minute
   Version: 2022-01-08T03:11:54Z
   Network: 2/2 OK
   Drives: 4/4 OK
```
with errors in the log saying it can't find the 'offline' test1 instance.

## Possible Solution
Not sure, could just be a docs issue.  Basically, you need to add the port used, in this case :9000 to the URLs of the cmd line.

`server --address :9000 https://minio-test1.test.local:9000/data{1...4} https://minio-test2.test.local:9000/data{1...4}`

I  can see why it happens.  Basically, even though you have told minio to use :9000 and HTTPS/TLS...if you don't include the port in the URLs, server pool 1 will try to 'reach' all those servers on :443.

But you can't use _http_ in the URL of the cmd line because you'll get, so you need to set _https_

`ERROR Unable to start the server: HTTP specified in endpoints, but the server in the local machine is configured with a TLS certificate
      > Please remove the certificate in the configuration directory or switch to HTTPS`

## Steps to Reproduce (for bugs)
1. Create 2 machines, with 4 DIRs each
2. Start the 1st Server Pool on machine 1:

server --address :9000 https://minio-test1.test.local/data{1...4}

3. Now restart the minio on machine 1 and machine 2, both with:

server --address :9000 https://minio-test1.test.local/data{1...4} https://minio-test2.test.local/data{1...4}

## Context
Basically, server pool 1 machines can't fine server pool 2 because by default, when using https it goes to port 443.  If the console is running on a different port.

## Regression
Not Sure

## Your Environment

* Version used (`minio --version`): appears to be in versions that I tested:  

    minio_version: 'RELEASE.2021-12-20T22-07-16Z'
    minio_version: 'RELEASE.2022-01-08T03-11-54Z'

* Server setup and configuration: Running inside Docker Container
* Operating System and version (`uname -a`): Docker on Centos 7, x68_64

**Comment by harshavardhana** [comment]
> `server --address :9000 https://minio-test1.test.local:9000/data{1...4} https://minio-test2.test.local:9000/data{1...4}`

First of all, this is a bad way to set up this is not what we support you shouldn't do this and this is not expected to work - however we allow and we should fix the underlying display issue. 

First of all, you should setup 

```
https://minio-test{1...2}.test.local:9000/data{1...4}
```

This is the expected distributed style.

**Comment by ming12713** [comment]
> > `server --address :9000 https://minio-test1.test.local:9000/data{1...4} https://minio-test2.test.local:9000/data{1...4}`
> 
> First of all, this is a bad way to set up this is not what we support you shouldn't do this and this is not expected to work in any meaningful way - however we allow it for testing - we should fix the underlying display issue.
> 
> Second redo your setup to look like this.
> 
> ```
> https://minio-test{1...2}.test.local:9000/data{1...4}
> ```
> 
> This is the expected distributed style.

@harshavardhana cloud you help me fix #14072 ,my cluster status is ok ,put/get object ok ,but can't reset config, thanks!

**Comment by ming12713** [comment]
> > > `server --address :9000 https://minio-test1.test.local:9000/data{1...4} https://minio-test2.test.local:9000/data{1...4}`
> > 
> > 
> > First of all, this is a bad way to set up this is not what we support you shouldn't do this and this is not expected to work in any meaningful way - however we allow it for testing - we should fix the underlying display issue.
> > Second redo your setup to look like this.
> > ```
> > https://minio-test{1...2}.test.local:9000/data{1...4}
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > This is the expected distributed style.
> 
> @harshavardhana cloud you help me fix #14072 ,my cluster status is ok ,put/get object ok ,but can't reset config, thanks!

>@harshavardhana thanks ,i had fixed #14072 issues,
follow is my steps:
```
* mc admin config export minio/ > minio`date +%Y%m%d%H%M`
* delete alldisk/.minio.sys/config/config.json
* mc admin service restart minio/
* mc admin config set minio/ notify_amqp:1 exchange="minio" exchange_type="fanout"  url="amqp://xxxx:xxxx@10.0.0.x:xxxx" delivery_mode="0" routing_key="minio"
```

**Comment by klauspost** [comment]
@ming12713 Please stop sending offtopic messages, or I will have you blocked completely.

**Comment by aglahe** [comment]
I'm confused as to why this is a bad setup?  Isn't this how you add more machines/disks to a cluster?

Maybe there is some confusion...this is just my test setup.  I have a "real" minio cluster with a bunch of nodes/disks and have one server pool that's been running for a while.  I'm about to add a second server pool of nodes/disks.  

Basically, I was testing adding a server pool to an existing minio cluster...and noticed that if you use TLS, and you don't explicitly set the port, it defaults to :443 because of HTTPS. 

Or, like you say, not a supported setup...how do we add nodes/disks to an existing TLS server pool? I was using: https://docs.min.io/minio/baremetal/installation/expand-minio-distributed.html as a guide.

**Comment by klauspost** [comment]
@aglahe That is also very different from what you wrote up top.

It is fine, and yes, you must specify the port. I don't think we can change this behavior as it can potentially break existing installs.

**Comment by aglahe** [comment]
Yep..sorry, figured it was easier to show "test steps to reproduce", should have added more description to the underlying setup. And agreed...not sure this was a 'bug' in code, but maybe a docs issue (wasn't a "documentation" tag for github issues)

Thanks for the confirmation

**Comment by ravindk89** [comment]
@harshavardhana unless you have other intentions, i'm going to transfer this to minio/docs and work on it there
